client/src/routes/game.jsx

```diff
@@ -26,12 +27,18 @@ const Game = () => {
   return (
     <div className='flex justify-center overflow-hidden select-none'>
       <Starfield starCount={5000} starColor={[255, 255, 255]} speedFactor={0.005} />
+      <div className='absolute top-0 right-0 w-auto max-h-100 z-50'>
+        <Menu data={data}/>
+      </div>
+      <div className='absolute top-0 w-2/3 sm:w-5/6 sm:max-w-xl z-50'>
```

Comment:
```suggestion
      <div className='absolute top-0 w-full sm:w-5/6 sm:max-w-xl z-50'>
```
Da går den helt ut i kanten når man er på telefon